commands/upgrade/util.py

```diff
@@ -8,11 +9,30 @@
 from leapp.config import get_config
 from leapp.exceptions import CommandError
 from leapp.repository.scan import find_and_scan_repositories
+from leapp.utils import audit
 from leapp.utils.audit import get_connection, get_checkpoints, get_messages
 from leapp.utils.output import report_unsupported
 from leapp.utils.report import fetch_upgrade_report_messages, generate_report_file
 
 
+def disable_database_sync():
+    def disable_db_sync_decorator(f):
+        @functools.wraps(f)
+        def wrapper(*args, **kwargs):
+            try:
+                saved = os.environ.get('LEAPP_DEVEL_DATABASE_SYNC_OFF', None)
```

Comment:
Nit: os.environ.get('LEAPP_DEVEL_DATABASE_SYNC_OFF') should return None by default

Reply:
Yes it does, but it is more obvious to the reader what you expect it to return.